python/langsmith/client.py

```diff
@@ -3412,6 +3412,43 @@ def list_examples(
             if limit is not None and i + 1 >= limit:
                 break
 
+    # dataset_name explicitly not supported to avoid extra API calls.
+    def search_examples(
```

Comment:
could rename it "few_shot_search" or something to make it more clear that the query is inputs and the output is meant to be injected into a prompt (ie the examples are bare, missing metadata that full Examples have)

Reply:
makes sense tor me, but also defer to @hinthornw for approval